client/my-sites/checkout/checkout/checkout.jsx

```diff
@@ -546,6 +546,11 @@ export class Checkout extends React.Component {
 			return false;
 		}
 
+		// Don't render when we're renewing a plan. Stick with the current period.
+		if ( planInCart.product_slug === currentPlanSlug ) {
+			return false;
+		}
+
```

Comment:
Does this prevent downsizing as well as upsizing your plan length? Is this what we want for all users or just business / monthly?

Reply:
This applies to all plans renewals.

It's already disabled for other plans/lengths, because you end up with ` availableTerms.length < 2` - see  #28306.

This is added here explicitly because the monthly term makes `availableTerms.length` be enough, but that ntroduces the issues I've mentioned in #28306.